libs/geography/src/facades/geography/geography.facade.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Action, Store, select } from '@ngrx/store';
+import { Dictionary } from '@ngrx/entity';
+
+import { DaffStoreFacade } from '@daffodil/core';
+
+import { DaffGeographyFeatureState } from '../../reducers/public_api';
+import {
+  getDaffGeographySelectors
+} from '../../selectors/public_api';
+import { DaffCountry } from '../../models/country';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffGeographyFacade<T extends DaffCountry> implements DaffStoreFacade<Action> {
+  loading$: Observable<boolean>;
+  errors$: Observable<string[]>;
+
+  countries$: Observable<T[]>;
+  ids$: Observable<(string | number)[]>;
```

Comment:
I think this should be `countryIds$`, since the facade is called `DaffGeographyFacade` instead of `DaffCountryFacade`

Reply:
good call, changed